docs/ref/unicode.txt

```diff
@@ -156,13 +134,11 @@ Django provides some assistance.
 * The function :func:`django.utils.encoding.iri_to_uri()` implements the
   conversion from IRI to URI as required by the specification (:rfc:`3987#section-3.1`).
 
-* The functions ``django.utils.http.urlquote()`` and
-  ``django.utils.http.urlquote_plus()`` are versions of Python's standard
-  ``urllib.quote()`` and ``urllib.quote_plus()`` that work with non-ASCII
-  characters. (The data is converted to UTF-8 prior to encoding.)
+* The functions ``urllib.parse.quote()`` and ``urllib.parse.quote_plus()``
```

Comment:
Before the  list, it says, "Django provides some assistance" but these are Python's functions, not Django's. They could be linked using `:meth:`.